hail/src/test/scala/is/hail/expr/ir/EmitStreamSuite.scala

```diff
@@ -1,14 +1,13 @@
 package is.hail.expr.ir
 
-import is.hail.annotations.{Region, SafeRow, ScalaToRegionValue, RegionValue}
-import is.hail.asm4s._
+import is.hail.annotations.{Region, RegionValue, SafeRow, ScalaToRegionValue}
+import is.hail.asm4s.{AsmFunction1, AsmFunction3, Code, GenericTypeInfo, MaybeGenericTypeInfo, TypeInfo}
```

Comment:
IntelliJ, even after cache invalidation and restart, complains about the presence of A.java in the test suite, so to prevent IntelliJ from failing to build, need to manually import here.

<img width="1371" alt="Screenshot 2019-12-16 11 12 45" src="https://user-images.githubusercontent.com/5543229/70923222-8b230c80-1ff5-11ea-9df0-fa03c22d9025.png">

Reply:
ah, thanks for fixing. Was going to PR separately.